global.json

```diff
@@ -1,6 +1,10 @@
 {
   "tools": {
-    "dotnet": "3.0.101"
+    "dotnet": "3.1.100"
+  },
+  "sdk": {
```

Comment:
Do we have to pin to a particular SDK?

Reply:
You need at least `3.1` to build `3.1`. adding `"rollForward": "major"` should enable any newer sdk to be used without issue.